docs/userGuide/contentAuthoring.md

```diff
@@ -589,12 +589,12 @@ Note:
 - Footers should not be nested in other components or HTML tags. If found inside, they will be shifted outside to be rendered properly.
 - [MarkBind components](#use-components) and [includes](#include-contents) are not supported in footers.
 
-### Using Anchors
+### Heading Anchors
 
-Sometimes, you may want a user of your site to be able to obtain a link to a heading within a page, to share with someone else for example.
+Your site's readers may want to obtain a link to a heading within a page, to share with someone else for example.
 
-MarkBind can automatically generate heading anchors for your page.
+MarkBind automatically generates heading anchors for your page.
 
-When a user hovers over a heading, a small anchor icon will become visible <i class="fas fa-anchor"></i>. Clicking this icon will redirect the page to that heading, producing a URL in the URL bar.
+When the viewer hovers over a heading in your page, a small anchor icon <i class="fas fa-anchor"></i> will become visible next to the heading. Clicking this icon will redirect the page to that heading, producing the desired URL in the URL bar that the viewer can share with someone else. Try it with the headings on this page!

```

Comment:
Let's standardize everything to 'readers', instead of having both 'viewer' and 'readers'. :P

Reply:
Done